pint_server/pint_db_migrate/versions/528cb85d6ad3_add_ipv6_column.py

```diff
@@ -0,0 +1,32 @@
+"""Add IPv6 column
+
+Revision ID: 528cb85d6ad3
+Revises:
+Create Date: 2021-07-30 19:05:31.695486
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '528cb85d6ad3'
+down_revision = None
```

Comment:
This will probably need to be updated once we have all the PRs lined up. Otherwise, the changes looks good!

Reply:
yep, I thought about that all the PRs but the first would have to keep that value on track, will update when the value for this PR is clear

Reply:
Done.